### PR TITLE
rock 2a: add usb more support and modify otg compatible to unknown

### PR DESCRIPTION
**Pull request by lsj123lsjlsj**
rock 2a: add usb more support and modify otg compatible to unknown

Signed-off-by: SongJun Li <lisongjun@radxa.com>

### CODE REVIEW

File: arch/arm64/boot/dts/rockchip/overlays/rk3528-dwc3-host.dts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// ROCK 3C USB OTG
```
Comment (RadxaYuntian) [inline arch/arm64/boot/dts/rockchip/overlays/rk3528-dwc3-host.dts:1]:
```suggestion
```

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]
另外commit标题改短一点，现在这个太长了

File: arch/arm64/boot/dts/rockchip/overlays/Makefile [hunk 2]
```diff
@@ -105,8 +105,10 @@ dtb-$(CONFIG_CPU_RK3399) += \
 	rock-4-radxa-display-10hd.dtbo
 
 dtb-$(CONFIG_CLK_RK3528) += \
-	rk3528-enable-tve.dtbo \
```
Comment (RadxaYuntian) [inline arch/arm64/boot/dts/rockchip/overlays/Makefile:108]: 这个删错了吧？

Comment (lsj123lsjlsj) [reply arch/arm64/boot/dts/rockchip/overlays/Makefile:108]: 只是按照顺序排到了后面 之前的顺序错了

**Review by RadxaYuntian — APPROVED** [review]

**Pull request merged**